larky/src/test/resources/quick_tests/test_fd_hmac_sha256_signature.star

```diff
@@ -0,0 +1,31 @@
+load("@stdlib//base64", base64="base64")
+load("@stdlib//json", json="json")
+load("@vendor//Crypto/Hash/HMAC", HMAC="HMAC")
+load("@vendor//Crypto/Hash/SHA256", SHA256="SHA256")
+
+# the following is only for tests (do not put in your yaml in live/prod)
+load("@stdlib//unittest", "unittest")
+load("@vendor//asserts", "asserts")
+
+
+def _test_fd_hmac_sha256_signatur():
+    API_KEY = b"1234"
+    CLIENT_REQUEST_ID = b"5678"
+    TIMESTAMP = b"1628108528"
+    PAYLOAD = {"one": 1, "two": 2}
+    API_SECRET = b"secretsauce"
+    msg = API_KEY + CLIENT_REQUEST_ID + TIMESTAMP + bytes(json.dumps(PAYLOAD), encoding='utf-8')
+    m = HMAC.new(API_SECRET, msg=msg, digestmod=SHA256)
+    output = m.digest()
+    print(output.hex())
+    print("base64 encoded: ", base64.b64encode(bytes(output.hex(), encoding='utf-8')))
```

Comment:
don't you need an assert?

Reply:
Oh yeah we do need one. This test came in from CS. Will update with an assert when I get a second

Reply:
Resolve: https://github.com/verygoodsecurity/starlarky/pull/163/commits/4bb18b93bdb38b50803a45a5a017a70d263b0273#diff-30d3fe2bfa551a69ccff0a6243b5a18910fadc1826335083a957e976168f1398R11-R29